internal/commands/commands.go

```diff
@@ -76,4 +77,21 @@ var (
 			},
 		},
 	}
+
+	Secrets = cli.CommandDefinition{
+		Use:         "secrets",
+		Aliases:     []string{"secret"},
+		Description: "Manage the secrets of your MongoDB Realm app",
+		Help:        "secret", // TODO(REALMC-7429): add help text description
+		SubCommands: []cli.CommandDefinition{
+			cli.CommandDefinition{
+				Use:         "list",
+				Aliases:     []string{"ls"},
+				Description: "List the names of secrets in your Realm app",
+				// TODO(REALMC-7429): add help text description
+				Help:    "Displays a list of data tables. Each data table displays the Name and ID of one (1) secret.",
```

Comment:
check out [here](https://docs.google.com/document/d/14fBZaN47HRe3-2Pk1uSfYcCxX3Dd95wK0tprS3WE1vk/edit) for better starter values with `Help` and `Description`

Reply:
The file should have the text from the linked google doc. I wasn't sure if the text was final and I can remove the TODO if so (left it as a placeholder for now)

Reply:
I see...yeah feel free to remove the TODO then, it's the best we got for now.  thanks!